src/components/StoriesGroup/components/StoriesPreview/StoriesPreview.tsx

```diff
@@ -4,13 +4,51 @@ import {MediaRenderer} from '../../../Stories/components';
 
 import {StoriesGroupItem} from '../../types';
 import {Chevron} from '../../../icons/Chevron';
-import {Button} from '../../../Button';
+import {Button, ButtonProps} from '../../../Button';
 
 import {block} from '../../../utils/cn';
 import './StoriesPreview.scss';
+import {ModalCloseReason} from 'src/components/Modal';
+import {StoriesItemMedia} from 'src/components/Stories/types';
```

Comment:
These imports won't work, they should be relative

Reply:
why this imports do not work ?

https://github.com/gravity-ui/uikit/blob/6c37dbe17c8802f06925f6c2fd276505cd87fad3/src/components/StoriesGroup/components/StoriesPreview/StoriesPreview.tsx#L25

https://github.com/gravity-ui/uikit/blob/6c37dbe17c8802f06925f6c2fd276505cd87fad3/src/components/StoriesGroup/components/StoriesPreview/StoriesPreview.tsx#L175

Reply:
Roots are not configured in the project. This might work for storybook, but not when code is compiled.

Reply:
[3377e1a](https://github.com/gravity-ui/uikit/pull/446/commits/3377e1a48b8feed7b0a2af732a4650ad39b6a80a)